misc/test_site/mac_addrs/9a02571e8f01/module_config.json

```diff
@@ -24,13 +24,81 @@
   "servers": {
     "tcp": {
       "ports": {
+        "20": {
+          "allowed": false,
+          "reason": "File Transfer Protocol (FTP) Server Data Transfer"
```

Comment:
this is not a reason, it's a description of what it is. I don't think that's appropriate here, since that as universal for what port 20 is, not something about this device. I would just take all these reason lines out... it's gonna be highly redundant.

Reply:
OK, I understand what you mean.
Do you think then that the test operator is expected to write individually the reason why that port is open? 
I lean towards thinking that changing "reason" to "service_name" or "description" would work best. Thoughts?

Reply:
I think these should be "description" -- and yes, somebody somewhere will have to include a "reason" or "justification" for why a port is open.  The ideal case is that everything is closed, so anything else will just require extra work.

Reply:
OK, I'll work on adding the field "description" then.
We could keep the "reason" field if you think it's the right place where to be. 
I'm not sure if it is going to be successfully edited by anyone, but we can keep for now and try if the workflow makes sense.